source/nodejs/adaptivecards/src/card-elements.ts

```diff
@@ -3647,6 +3675,10 @@ export class TextInput extends Input {
             result = document.createElement("input");
             result.className = this.hostConfig.makeCssClassName("ac-input", "ac-textInput");
             result.type = Enums.InputTextStyle[this.style].toLowerCase();
+
+            if (this.inputStyle !== null && this.inputStyle === Enums.InputStyle.ReadWrite && !this.inlineAction) {
```

Comment:
The SDK uses "undefined" across the board, and not null. The value of `this.inputStyle` will be read from the underlying property bag which will return `undefined` if `inputStyleProperty` doesn't have a value. Thus the check for `this.inputStyle !== null` is unnecessary and will always return `true`. Either change it to `this.inputStyle !== undefined` or remove it altogether (it is unnecessary).